components/grid.css

```diff
@@ -0,0 +1,171 @@
+/* grid */
+
+/* 2 col grid */
+
+/* 50 50 grid */
+
+.grid-2 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+.grid-2 > :nth-child(1) {
+  grid-column: 1/2;
+}
+.grid-2 > :nth-child(2) {
+  grid-column: 2/3;
+}
+
+/* 30 70 grid */
+
+.grid-30-70 {
+  display: grid;
+  grid-template-columns: 30% 70%;
+}
+
+/* 70 30 grid */
+
+.grid-70-30 {
+  display: grid;
+  grid-template-columns: 70% 30%;
+}
+
+/* 3 col grid */
+
+.grid-3 {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+/* 4 col grid */
+
+.grid-4 {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+}
+
+/* grid 20 80  */
+
+.grid-20-80 {
+  display: grid;
+  grid-template-columns: 20% 80%;
+}
+
+/* media queries for grid only mobile */
+
+@media only screen and (max-width: 480px) {
+  .grid-4 {
+    display: grid;
+    grid-template-columns: 1fr;
+  }
+  .grid-30-70 {
+    display: grid;
+    grid-template-columns: 1fr;
+  }
+
+  .grid-70-30 {
+    display: grid;
+    grid-template-columns: 1fr;
+  }
+}
+
+/* navbar */
+
+.navbar {
+  background-color: white;
+  padding-left: 3%;
```

Comment:
Suggestion: you can specify all paddings using the short-hand property.

Reply:
Yes I will implement it right away